lib/core/crypto/eosdart/src/numeric.dart

```diff
@@ -89,7 +89,7 @@ Uint8List signedDecimalToBinary(int size, String s) {
   var result = decimalToBinary(size, s);
```

Comment:
This is the fix, everything else here is better errors and better logging since this bug kinda fell through and didn't produce any output.